e2e/helpers.ts

```diff
@@ -687,13 +687,10 @@ export async function getOnchainBalance(addy: string, contract: string) {
 export async function transactionStatus() {
   const provider = getDefaultProvider('http://127.0.0.1:8545');
   const blockData = await provider.getBlock('latest');
-  const txn = await provider.getTransaction(blockData.transactions[0]);
-  const txnData = txn.wait();
-
-  // transactionResponse.wait.status returns '1' if txn is successful
-  // it returns '0' if the txn is a failure
-  const txnStatus = (await txnData).status === 1 ? 'success' : 'failure';
-
+  const txnReceipt = await provider.getTransactionReceipt(
```

Comment:
idk if it matters in this testing env, but `txnReceipt` will be `null` if the tx is `pending`

Reply:
honestly fine as long as it returns 'failure' for this function